metricflow/sql/sql_exprs.py

```diff
@@ -944,8 +944,9 @@ class SqlWindowFunction(Enum):
     Values are the SQL string to be used in rendering.
     """
 
-    FIRST_VALUE = "first_value"
-    ROW_NUMBER = "row_number"
+    FIRST_VALUE = "FIRST_VALUE"
+    LAST_VALUE = "LAST_VALUE"
```

Comment:
do we need to include this?

Reply:
the last_value enum

Reply:
Oh yes! Sorry forgot to include in the PR description! Will also be used for non-default granularities with cumulative metrics. We could use `FIRST_VALUE` and order `desc`, but I decided this was more readable because it aligns with the YAML argument names (which are `first`/`last`).